src/node_jsvmapi.h

```diff
@@ -18,6 +18,7 @@
 #ifndef SRC_NODE_JSVMAPI_H_
 #define SRC_NODE_JSVMAPI_H_
 
+#include "node.h"
```

Comment:
Can inclusion of `node.h` be avoided? It would be better if all the node APIs (and V8 APIs included by `node.h`) were not available when building a NAPI module.